docs/mobile-apps/features/network-capture.md

```diff
@@ -0,0 +1,104 @@
+---
+id: network-capture
+title: Network Traffic Capture
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Network Traffic Capture is a functionality that gives you the ability to record network traffic and HTTP/HTTPS request headers during your native Android and iOS/iPadOS mobile app app tests. This provides granular insight into your app’s behavior, facilitates debugging, and helps you determine which development team is responsible for application failure.
+
+With this feature, traffic is recorded on our side and exposes the HAR file through your test’s endpoint, producing HTTP traffic information to help you troubleshoot.
+
+
+## What You'll Need
+
+* A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up))
+* A native Android, iOS, or iPadOS mobile app that makes HTTP/HTTPS requests. We currently support the following:
+   * Android 6.0 and higher
+   * For Swift: iOS 11.0 and higher
+   * For Objective-C: iOS (all)
+
+
+## Using Network Traffic Capture
+
+To enable network traffic capturing in your tests:
+
+1. On Sauce Labs, click **Live** > **Mobile App**.
+2. [Upload your mobile app to Sauce Labs](/mobile-apps/app-storage) through the UI or CLI.
```

Comment:
should we add a reference link here to the actual CLI upload? how they can do it? https://docs.saucelabs.com/mobile-apps/app-storage/#uploading-apps-via-rest-api